components/identity.oauth.uma.permission.endpoint/src/gen/java/org/wso2/carbon/identity/oauth/uma/permission/endpoint/PermissionApi.java

```diff
@@ -29,9 +31,9 @@
         
         @io.swagger.annotations.ApiResponse(code = 400, message = "Bad Request") })
 
-    public Response requestPermission(@ApiParam(value = "The requested permissions." ) ResourceModelDTO requestedPermission)
+    public Response requestPermission(@ApiParam(value = "The requested permissions."  ) ResourceModelDTO requestedPermission, @Context MessageContext context)
     {
-    return delegate.requestPermission(requestedPermission);
+    return delegate.requestPermission(requestedPermission,context);
```

Comment:
Formatting issue. Use wso2 code style.

Reply:
Since this is auto generated it was decided to keep it as it is.